File: 1550-three-consecutive-odds/1550-three-consecutive-odds.py
class Solution(object):
    def threeConsecutiveOdds(self, arr):
        """
        :type arr: List[int]
        :rtype: bool
        """
        count = 0
        for i in arr :
            if(i % 2  == 1):
                count += 1
                if(count == 3):
                    return True
            else:
                count = 0
        return False

File: 1550-three-consecutive-odds/1550-three-consecutive-odds.c
bool threeConsecutiveOdds(int* arr, int arrSize) {
    int countor = 0 ;
    for(int i = 0 ; i < arrSize ; i++) 
    {
        if(arr[i] % 2 == 1) countor++ ; 
        else countor = 0 ; 
        if (countor == 3) return true ; 
    }
    return false ; 
}